site/examples/fixed_header.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { HtmlExample } from '../html_example';
+
+const fixed_header = `<div class='bg-darken10 viewport-half'>
+<!-- In practice, it makes sense to use this for the container instead:
```

Comment:
The fixed and not-fixed versions of this header should be as similar as possible. You could use:

`<div class='absolute top right bottom left scroll-auto'></div>`

as an alternative to 

`<div class='fixed top right bottom left scroll-auto'></div>`

Only one class different, and should communicate the idea just as well.

---

Also I don't think you need `scroll-auto` on this example.